releasenotes/notes/fix-dd-trace-methods-2c1b4af30a7bf3ef.yaml

```diff
@@ -0,0 +1,5 @@
+---
+deprecations:
```

Comment:
I think we could make this two notes, one is a feature with the new notation, and one is the deprecation.

Reply:
Perhaps we could reference the docs in the feature rn?